### PR TITLE
 Swap two numbers without using the third variable.

### PR DESCRIPTION
**Pull request by Mayur-Suryawanshi1992**
Swap two numbers without using the third variable.

### CODE REVIEW

File: JavaAssignments/src/mayur/SwapNumberWithoutThirdVeriable.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package mayur;
+
+public class SwapNumberWithoutThirdVeriable 
+{
+	public static void main(String[] args) 
+	{
+		float Num1 = (float) Double.parseDouble(args[0]);
```
Comment (mkanani13) [inline JavaAssignments/src/mayur/SwapNumberWithoutThirdVeriable.java:7]: Please use Float.parseFloat(args[0]).
Approving your request.

Comment (mkanani13) [reply JavaAssignments/src/mayur/SwapNumberWithoutThirdVeriable.java:7]: rest Code looks good to me.

**Review by mkanani13 — APPROVED** [review]

File: JavaAssignments/src/mayur/SwapNumberWithoutThirdVeriable.java (hunk 1 above)
Comment (Mayur-Suryawanshi1992) [reply JavaAssignments/src/mayur/SwapNumberWithoutThirdVeriable.java:7]: Used Float.parseFloat(args[0]) in other programs

**Review by mkanani13 — APPROVED** [review]
Code looks good to me. Keep going Mayur.

**Pull request merged**